src/mongoose_scram.erl

```diff
@@ -74,18 +70,16 @@ enabled(HostType, cyrsasl_scram_sha512_plus) -> is_password_format_allowed(HostT
 enabled(_HostType, _Mechanism) -> false.
 
 is_password_format_allowed(HostType, Sha) ->
-    case mongoose_config:get_opt([{auth, HostType}, password_format], scram) of
-        plain -> true;
-        scram -> true;
-        {scram, ConfiguredSha} -> lists:member(Sha, ConfiguredSha)
+    case mongoose_config:get_opt([{auth, HostType}, password]) of
+        #{format := scram, hash := ConfiguredSha} -> lists:member(Sha, ConfiguredSha);
+        #{format := _PlainOrScram} -> true
```

Comment:
Nice, that's very nice, good optimisation against the common case, less `lists:member/2` 👏🏽